### ISSUE TITLE
Accessibility: Detail views use headings for styling purposes

### ISSUE THREAD

**Issue by sedlatschek** [laravel/nova-issues#3431]
- Laravel Version: 8.44.0
- Nova Version: 3.26.1
- PHP Version: 7.4.6
- Database Driver & Version: MySQL 5.7
- Operating System and Version: Win10
- Browser type and version: Chrome 88

### Description:
Headings are very important for accessibility. Screenreaders can be used to only read out headings, allowing the user the navigate freely between them. Headings are only expected to be used above content.

Detail views contain h4 headings for styling purposes. This is confusing to blind users.

![image](https://user-images.githubusercontent.com/58665536/121053383-93f43900-c7bb-11eb-8cc1-4b74d03cb7ee.png)

Laravel Nova is not accessible. This is a problem in enterprise environments, where accessibility is a must requirement.

**Comment by davidhemphill** [comment]
👋 Hi there! In order to keep this repository focused on bug reports, we auto-close feature requests and requests for help. Feel free to post your feature requests so others can discuss and add reactions. We'll keep an eye on them for later planning.
